views/topbar_backend.php

```diff
@@ -3,21 +3,22 @@
         <div class="nav-wrapper">
             <a href="/accueil-de-jerome" class="brand-logo">Accueil</a>
             <?php
-            if($page != 'login' && $page != 'new' && $page != 'password'){
+
+            if($_GET['page']!= 'login' && $_GET['page']!= 'new' && $_GET['page']!= 'password'){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Detected usage of a possibly undefined superglobal array index: $_GET['page']. Use isset() or empty() to check the index exists before using it](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)